LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2012-2019 Vanderbilt University, ISIS
+Copyright (c) 2012-2022 Vanderbilt University, ISIS
```

Comment:
I know it is not 2023 right now, but maybe it would be wiser to put it there ahead of time, so we (I) do not forget about it :)